_translator-files/po/sv-SE/FAQ.po

```diff
@@ -70,165 +71,167 @@ msgid ""
 msgid "If somebody lists a Server on one of the Directories built into Jamulus, they accept that anyone can play on it. Jamulus has no password protection or other authentication mechanisms. Whilst some Servers may state their policies in the welcome message you will see in the chat window, Jamulus itself does nothing to enforce these."
-msgstr ""
+msgstr "Om någon listar en server på en av de kataloger som ingår i Jamulus, accepterar de att vem som helst kan spela på den. Jamulus har ingen lösenordsskydd eller andra autentiseringsmekanismer. Även om vissa servrar kan ange sina policyer i välkomstmeddelandet du ser i chattfönstret, gör Jamulus i sig inget för att verkställa dessa."
 
 #. type: Plain text
 #: ../wiki/en/FAQ.md:34
 #, markdown-text
 msgid "Note also that Servers do not have to be registered on a Directory in order for Jamulus Clients to connect to them. Server operators can just give out the address of their Servers to those they want to play with, and they can then connect directly. Have a look at [this overview for more information](/wiki/Running-a-Server#server-modes)."
-msgstr ""
+msgstr "Observera också att servrar inte behöver vara registrerade i en katalog för att Jamulus-klienter ska kunna ansluta till dem. Serveroperatörer kan helt enkelt ge ut adressen till sina servrar till de vill spela med, och de kan då ansluta direkt. Ta en titt på [denna översikt för mer information](/wiki/Running-a-Server#server-modes)."
 
 #. type: Title ###
 #: ../wiki/en/FAQ.md:35
 #, markdown-text, no-wrap
 msgid "Why shouldn’t I use wireless equipment?"
-msgstr ""
+msgstr "Varför bör jag inte använda trådlös utrustning?"
 
 #. type: Plain text
 #: ../wiki/en/FAQ.md:38
 #, markdown-text
 msgid "Whether or not you will be able to play in time with other musicians depends mainly on how much latency (delay) you have in your sound signal and whether you are all following [The Golden Rule](/wiki/Client-Troubleshooting#you-all-sound-ok-but-its-difficult-to-keep-together). An overall delay much bigger than 50 ms usually makes it too hard to play in time unless you train yourself to do so. Some musicians report they can play in time with delays of up to 70 ms by playing ahead of the beat. But much beyond that and it becomes too difficult unless timing isn’t an important factor in the music or you just want to listen."
-msgstr ""
+msgstr "Om du kan spela i takt med andra musiker beror främst på hur mycket latens (fördröjning) du har i ditt ljudsignal och om ni alla följer [Den gyllene regeln](/wiki/Client-Troubleshooting#you-all-sound-ok-but-its-difficult-to-keep-together). En total fördröjning som är mycket större än 50 ms gör det vanligtvis för svårt att spela i takt, om du inte tränar dig själv att göra det. Vissa musiker rapporterar att de kan spela i takt med fördröjningar på upp till 70 ms genom att spela före takten. Men mycket mer än så blir det för svårt om timing inte är en viktig faktor i musiken eller om du bara vill lyssna."
```

Comment:
Translate text after # in link.

Reply:
`ni-låter-alla-okej-men-det-är-svårt-att-hålla-ihop`